tools/nxfuse/README_SMARTFS.md

```diff
@@ -1,20 +1,85 @@
-nxfuse
-======
+# How to make smartfs user image for TizenRT
+
+## Contents
+> [How to Enable SmartFS user image](#how-to-enable-smartfs-user-image)  
+> [About NxFuse tool](#about-nxfuse-tool)  
+> [How to use nxfuse](#how-to-use-nxfuse)  
+> [Formatting and Reformatting](#formatting-and-reformatting)  
+
+## How to Enable SmartFS user image
+SMARTFS file system(CONFIG_FS_SMARTFS) needs to be enabled in order to use SMARTFS image.
+
+Enable FS_SMARTFS config through menuconfig at *os* folder.
+
+### Configurations
+Enable below configurations as followings
+
+```
+->BuildSetup
+	->Binary Output Formats
+		-> Enable "Make smartfs binary image" (=y)
+```
+
+```
+->Board Selection
+	-> Adjust smartfs partition size
+```
+
+```
+->File Systems
+	-> Memory Technology Device (MTD) Support
+		-> SMART Device options
+			->  set "SMART Device sector size" = (512 bytes)
```

Comment:
Do we need to change this with 512 bytes for all of boards? Can't we use the value set already?

Reply:
No need to change for all boards, some of the boards have this size wrongly quoted. For ex: imxrt1020 config has 1024. which wrong, dont know why its been quoted like that.

thats the reason, we have added this as a caution note.

Reply:
Let's say 512 Bytes is an example on imxrt. It looks that value is mandatory to make smartfs image.

Reply:
Removed this part and added change to fetch the size from .config